### DISCUSSION TITLE
reinstate contributions tracking from "mirror" repositories

### DISCUSSION THREAD

**Discussion by pixelb** [community/community#14974] [Other Feature Feedback, Questions, & Ideas]
It used to be the case if you were an admin or member of an upstream _mirror_ repository,
that contributions you made there would be considered in ones contribution view.
I can see that forked repos should not be considered as one could fork a repo and pad commits,
however upstream mirrors are a small curated set, and it would be quite useful to
get user contributions from non github repos into the "social graph".

As a concrete example I'm an admin of the https://github.com/coreutils/coreutils/ mirror,
but none of my contributions there appear on my timeline

**Comment by z-aki** [comment]
+1

**Comment by pixelb** [comment]
It looks like tracking in "mirror" repos has been restored

**Comment by z-aki** [comment]
maybe only because you're a "member" of the repo.
